patches/net/minecraft/network/protocol/BundlerInfo.java.patch

```diff
@@ -0,0 +1,68 @@
+--- a/net/minecraft/network/protocol/BundlerInfo.java
++++ b/net/minecraft/network/protocol/BundlerInfo.java
+@@ -23,7 +_,7 @@
+     };
+ 
+     static <T extends PacketListener, P extends BundlePacket<T>> BundlerInfo createForPacket(
+-        final Class<P> p_265438_, final Function<Iterable<Packet<T>>, P> p_265627_, final BundleDelimiterPacket<T> p_265373_
++        final Class<P> p_265438_, final Function<Iterable<Packet<? super T>>, P> p_265627_, final BundleDelimiterPacket<T> p_265373_
+     ) {
+         return new BundlerInfo() {
+             @Override
+@@ -38,11 +_,31 @@
+                 }
+             }
+ 
++            @Override
++            public void unbundlePacket(Packet<?> bundlePacket, Consumer<Packet<?>> packetSender, io.netty.channel.ChannelHandlerContext context) {
++                if (bundlePacket.getClass() == p_265438_) {
++                    P p = (P)bundlePacket;
++                    java.util.List<Packet<?>> packets = net.neoforged.neoforge.network.registration.NetworkRegistry.getInstance().filterGameBundlePackets(context, p.subPackets());
++                    if (packets.isEmpty()) {
++                        return;
++                    }
++                    if (packets.size() == 1) {
++                        packetSender.accept(packets.get(0));
```

Comment:
The delimiter is not accepted because the bundle is unpacked.
The purpose of a bundle is to process all packets within that bundle at the same time. If there is only one packet, having the bundle or not makes no difference to the result. As such, no delimiters are needed.